tools/release/README.md

```diff
@@ -1,4 +1,57 @@
-# Checkbox release process (Debian packages)
+# Checkbox release process
+
+> **_NOTE:_** Check the [Launchpad Builders status] before triggering a release.
+Sometimes, the builders will be down and this might prevent building the
+packages.
+
+## Bump the checkbox version
+Run the [Bumpversion workflow] (the default increment is **minor**)  
+Steps in this workflow:
+- Run [Bumpversion] for all checkbox sub-projects, snaps and debian packages
+- Open a new PR to approve and merge
+- Provide tagging instructions
+
+## Tag the release
+- Clone the repository once the PR is merged
+  ```
+  git clone git@github.com:canonical/checkbox.git
+  ```
+- Tag the release following the instructions given in the PR
+  ```
+  git tag -s "v2.4" -m "Bump version: 2.3 → 2.4"
+  ```
+- Push the tag to origin
+  ```
+  git push --tags
+  ```
+  > **_NOTE:_** Having to clone and not push the tag from the bumpversion workflow is actually a Github action limitation[^1]:
+  > *if a workflow run pushes code using the repository's GITHUB_TOKEN, a new workflow will not run even when the repository contains a workflow configured to run when push events occur.*
+
+## Monitor the build and publish workflows
+3 workflows are triggered on tag push events:
+- [checkbox deb packages] *(built and published to the testing PPA)*
+- [checkbox snap packages] *(built and uploaded to their respective beta channels)*
+- [checkbox core snap packages] *(built and uploaded to their respective beta channels)*
+
+In addition to the above workflows, a draft release is created on Github with
+an auto-generated changelog.
+
+Check the related Github Action logs to see if everything runs as expected:
+
+- snapcraft is not blocked during the snap build process. For example, in this 
+[build](https://github.com/canonical/checkbox/actions/runs/4371649401/jobs/7649877336), the i386 build was blocked on an error (`Chroot problem`) for hours 
```

Comment:
Not sure if you want to cap each line at 80 characters, in which case this one has to be adapted (you can put the link at the bottom of the document, for instance).